apps/chatbot/src/app/main.py

```diff
@@ -18,30 +18,30 @@
 
 params = yaml.safe_load(open("config/params.yaml", "r"))
 prompts = yaml.safe_load(open("config/prompts.yaml", "r"))
-chatbot = Chatbot(params, prompts)
-
 AWS_DEFAULT_REGION = os.getenv('CHB_AWS_DEFAULT_REGION', os.getenv('AWS_DEFAULT_REGION', None))
 
+chatbot = Chatbot(params, prompts)
+
 
 class Query(BaseModel):
   question: str
   queriedAt: str | None = None
 
-if (os.getenv('environment', 'dev') == 'local'):
-  profile_name='dummy'
-  endpoint_url='http://localhost:8000'
-  region_name = AWS_DEFAULT_REGION
-
 boto3_session = boto3.session.Session(
-  profile_name = locals().get('profile_name', None),
-  region_name=locals().get('region_name', None)
+  region_name=AWS_DEFAULT_REGION
 )
```

Comment:
in boto3 session are missing the `aws_access_key_id` and `aws_secret_access_key`